cloudformation/deploy/template.yaml

```diff
@@ -97,6 +97,11 @@ Conditions:
               - !Ref SubEnvironment
               - "authdev1"
 
+  UseMfaResetWithIpv:
+      Fn::Equals:
```

Comment:
The indentation here looks a bit off to me

Reply:
I've changed it a little. Both times it passed the cloudformation linter we use so I believe is valid, but whatever looks right to people I'm happy to go with